### ISSUE TITLE
[Bug][a11y]PiP doesn't interact with talkback using Google Pixel 3

### ISSUE THREAD

**Issue by sv-sdeiac** [mozilla-mobile/fenix#10387]
### Prerequisites
The talkback option is enabled from the Android settings.

## Steps to reproduce
1. Open a video in youtube.com;
1. Go to full-screen;
1. Display the Picture-in-picture video mode;
1. Try to interact with the PiP screen using 1 or 2 fingers.

### Expected behavior
The user is able to interact with the PiP mode.

### Actual behavior
The user can't interact with the PiP mode.

### Device information
Android device: Google Pixel 3 (Android 10).
Fenix version: 4/30 #21210606.
Not reproducible using LG G7 Fit (Android 8.1).
[Video](https://drive.google.com/file/d/1E41bI4HM2emTOxjWMjN-0dr3vng2XUhH/view?usp=sharing).

**Comment by softvision-miralobontiu** [comment]
Cannot activate Pip on Beta 79.0.0-beta.5, or the 7/11 Nightly build using TB gestures on Google Pixel (Android 10).
